content/blog/2024/anchor-position-polyfill.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Updates to the Anchor Position Polyfill
+sub: Catching up to the spec
+date: 2024-07-02
+image:
+  src: blog/2024/anchor1.jpg
+  alt: >
+    A rusty anchor hanging with the sea in the background.
+author: james
+oss: true
+tags:
+  - Article
+  - OSS
+  - Anchor Position
+  - CSS
+summary: |
+  Sponsor support for continued development of the
+  Anchor Position Spec is making the Anchor Position Polyfill more useful.
```

Comment:
I don't really understand this sentence. Who is sponsoring what? The spec is being sponsored?

Reply:
Changed to "Our sponsors are supporting the continued development of the CSS Anchor Positioning Polyfill. Here's a summary of the latest updates."